### DISCUSSION TITLE
public/build directory is getting bloated

### DISCUSSION THREAD

**Discussion by thomasverleye** [remix-run/remix#3156] [Q&A]
Hi again 👋 

When setting up the new project we discovered when running `remix dev`, the `public/build` directory get's bloated pretty quickly.
- A lot of new manifest files are being created when modifying files from the `app` directory.
- When imported css files change a new hashed version is being created.

I understand that this is something necessary when running `remix build` so when you (re-)deploy your project old version that are being cached on clients don't throw any errors.

But what is the benefit of creating all this clutter when running dev?
Can't there be an automated way of clearing out this directory on each `remix dev` run?

Or am I misunderstanding the point?

**Comment by kiliman** [comment]
I believe the next release will delete the previous build output when remix dev/watch starts.

In the meantime, I just add this to my *package.json* scripts:
```json
"clean": "rimraf build public/build",
"dev": "npm run clean && remix dev",
```
Add `rimraf` to your `devDependencies`

**Reply by thomasverleye** [reply]
Smart, thank you so much!

**Comment by thomasverleye** [comment]
I just want to point out that this isn't fixed just yet in the next release.
They actually fixed that indeed but then reverted it: https://github.com/remix-run/remix/commit/9cd670f8d9d46cfb8c0b4377bbf318c0eb9633cd

Here's an issue that is still open on that: https://github.com/remix-run/remix/issues/2432

You are still able to use the code above that @kiliman suggested 🙇
